server/bleep/src/lib.rs

```diff
@@ -141,14 +141,12 @@ impl Environment {
         }
     }
 
-    /// Use AAA layer (authentication, authorization, accounting)
     pub(crate) fn use_aaa(&self) -> bool {
-        use Environment::*;
-        match self {
-            Server => false,
-            InsecureLocal => false,
-            PrivateServer => true,
-        }
+        matches!(self, Self::PrivateServer)
+    }
+
+    pub(crate) fn serve_frontend(&self) -> bool {
+        matches!(self, Self::PrivateServer)
```

Comment:
Do we need two methods for the same thing here?

Reply:
@rsdy requested this, I'll revert the change